rpcs3/Emu/System.cpp

```diff
@@ -3177,12 +3132,16 @@ void Emulator::AddGamesFromDir(const std::string& path)
 
 		if (const auto error = BootGame(dir_path, "", false, true); error == game_boot_result::no_errors)
 		{
-			if (std::exchange(m_games_yml_invalidated, false))
```

Comment:
Why not log it?

Reply:
It's already logged in load

Reply:
Also, i would have to clear the dirty flag each time which only complicates things for no reason